client/src/pages/OnBoarding/index.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useRef } from "react";
+import { Paper, Grid, Button, makeStyles } from "@material-ui/core";
+
+import SetTimezoneUrl from "./SetTimezoneUrl";
+import ConnectGoogleCalendar from "./ConnectGoogleCalendar";
+import SetAvailability from "./SetAvailability";
+
+const OnBoarding = (props) => {
+  //styles
+  const classes = useStyles();
+
+  //data related variables
+  const [pageNum, setPageNum] = useState(0);
+  const url = useRef();
+  const timezone = useRef();
+  const days = useRef();
+  const startHour = useRef();
+  const finishHour = useRef();
+
+  const FlipToNextPage = () => {
+    //send data to backend here
+    switch (pageNum) {
```

Comment:
We could probably use an array of handler functions or something similar in place of a switch statement since we're already using integers for the cases. Not a big problem but may be cleaner.